project_name/local.example.env

```diff
@@ -0,0 +1,2 @@
+DJANGO_DEBUG=True
```

Comment:
What do you think about creating a folder called `contrib` on the root to store files like these? We're probably going to have more "example files", so maybe it's good to centralized them!